packages/ember-glimmer/lib/renderer.js

```diff
@@ -367,8 +369,10 @@ class Renderer {
 
     try {
       this._renderRoots();
-    } finally {
+    } catch (e) {
```

Comment:
The issue is with debugging the rethrow. It completely masks errors and makes "stop on uncaught" exceptions in chrome unusable. 

I was suggesting a way to avoid the catch, but still have the behavior you changed here (only update last revision in the error case).